### ISSUE TITLE
Intermittent UI test failure - < CrashReportingTest.useAppWhileTabIsCrashedTest>

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#24418]
### Stacktrace:
[Logcat](https://github.com/mozilla-mobile/fenix/files/8334440/useAppWhileTabIsCrashedTest.txt)

**Comment by AndiAJ** [comment]
Failed 1x on 3/23 Main 
[Firebase link](https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/7772509306765377746/executions/bs.d79351bdfbcef89f/testcases/1/test-cases)
[Firebase link](https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/7772509306765377746/executions/bs.706bf505a9f18417/testcases/1/test-cases)

**Comment by sv-ohorvath** [comment]
A11y issue causing the failure was fixed, can be reopened if it fails again.
